rpcs3/Emu/RSX/VK/vkutils/shared.h

```diff
@@ -1,12 +1,14 @@
 #pragma once
 
 #include "../VulkanAPI.h"
+#include <string>
 
 namespace vk
 {
 #define CHECK_RESULT(expr) { VkResult _res = (expr); if (_res != VK_SUCCESS) vk::die_with_error(_res); }
+#define CHECK_RESULT_EXT(expr, msg) { VkResult _res = (expr); if (_res != VK_SUCCESS) vk::die_with_error(_res, msg); }
 
-	void die_with_error(VkResult error_code,
+	void die_with_error(VkResult error_code, std::string message = {},
```

Comment:
I wonder if message can be a reference

Reply:
does it really matter if we are killing the whole program ?

Reply:
sure, it's nothing, just pretending i'm a pro

Reply:
Use const ref anyway, its just nicer.
const std::string& message = {}

Reply:
But I modify the message if not empty

Reply:
I'll rewrite it later to avoid that.